helpers/sdk_version.ts

```diff
@@ -0,0 +1,8 @@
+import {ensureExists} from './ensure';
+import semver from 'semver';
+import {version} from '../package.json';
```

Comment:
I think we initially did this much simpler and more correct approach for getting the version from package.json, but it wreaked havoc with our CI build-is-up-to-date checks for dependabot PRs. See the PR comments here for some more details: https://github.com/coda/packs-sdk/pull/831

So we probably want to go back to other approach.

Reply:
Reverted this code while removing `sdkVersion` from the metadata itself